src/mpyl/reporting/formatting/markdown.py

```diff
@@ -120,51 +124,48 @@ def execution_plan_as_markdown(run_result):
 
 
 def to_markdown_test_report(suites: list[TestSuite]):
+    print(suites)
```

Comment:
Debug print

Reply:
Resolved in https://github.com/Vandebron/mpyl/pull/185/commits/383c76e60e9b31de361c83c6a9fa445a39474ab2